src/components/layout/CyVerseAppBar.js

```diff
@@ -805,30 +784,10 @@ function CyverseAppBar(props) {
                     />
                 )}
             </div>
-            {runTour && (
-                <Joyride
-                    steps={getSteps(i18nTour)}
-                    run={runTour}
-                    showProgress={true}
-                    continuous={true}
-                    disableOverlayClose={true}
-                    callback={handleJoyrideCallback}
-                    stepIndex={tourStepIndex}
-                    styles={{
-                        options: {
-                            arrowColor: theme.palette.error.main,
-                            backgroundColor: theme.palette.error.contrastText,
-                            overlayColor: theme.palette.silver,
-                            primaryColor: theme.palette.error.main,
-                            textColor: theme.palette.info.main,
-                            zIndex: 10000000,
-                        },
-                        tooltipContainer: {
-                            textAlign: "left",
-                        },
-                    }}
-                />
-            )}
+            <ProductTour
+                onTourExit={() => setRunTour(false)}
+                runTour={runTour}
```

Comment:
I no longer see where this AppBar can set `runTour` to true.

Reply:
Its here - https://github.com/cyverse-de/sonora/blob/3efb99f8c190ae3eace17752e3fe153b9dae5afb/src/components/layout/CyVerseAppBar.js#L827
if am understanding your comment correctly.

Reply:
Ah, I see it now. Thanks. I didn't see it in the diff, and I didn't think to check the unchanged code 👍